Detectors/TPC/workflow/src/tpc-reco-workflow.cxx

```diff
@@ -139,6 +140,7 @@ WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
   auto nLanes = cfgc.options().get<int>("tpc-lanes");
   auto inputType = cfgc.options().get<std::string>("input-type");
   auto requireCTPLumi = cfgc.options().get<bool>("require-ctp-lumi");
+  auto lumiScaleMode = cfgc.options().get<bool>("corrmap-lumi-mode");
```

Comment:
```suggestion
  const auto lumiScaleMode = cfgc.options().get<bool>("corrmap-lumi-mode");
```

Reply:
But none of the other options are const'ed and also there are methods implemented which could technically allow a change of this variable. Should this still be const'ed?

Reply:
I would keep it identical for all of them. In principal you can make all of them const, but doesn't really matter.